presto-common/src/main/java/com/facebook/presto/common/RuntimeUnit.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.common;
+
+import com.facebook.drift.annotations.ThriftEnum;
+import com.facebook.drift.annotations.ThriftEnumValue;
+
+@ThriftEnum
+public enum RuntimeUnit
+{
+    NONE(0), NANO(1), BYTE(2);
```

Comment:
1. Can we have one more `TIME` to represent how many times 
2. Can we remove `NONE`, why do we need it?

Reply:
> Can we have one more TIME to represent how many times

You mean use `TIME` for the counting metrics like `driverCountPerTask ` or `fragmentResultCacheHitCount`? I think it might be a little be confusing. `TIME` looks like a time unit. Also the current setup is align with [Velox](https://github.com/facebookincubator/velox/blob/fdbbfd2354075da80e98f18055d744172f15e15f/velox/common/base/RuntimeMetrics.h#L25). I think it's good for start.

> Can we remove NONE, why do we need it?
 
I would suggest keeping it:
1. For compatibility reasons, we are introducing this unit to runtime metrics and there are many metrics haven't been set up with units yet, we would like those metrics to have a default fall-back value `NONE`.
2. For some metrics like `optimizedWithMaterializedView` ( _1 means the query is rewritten to read from available materialized views to improve the performance_), it does make much sense to set a unit to them, so let's keep it as `NONE`.

Reply:
Sounds good, can we 
1. Link the Velox file in the description as well since those two have to be compatible with each other
2. Can we add an issue or task as follow up to slowly deprecate the NONE and adopt meaningful units
3. As for `TIME`, I understand the confusion (I would add a comment to avoid confusion), but also hard to think of any other names :) , feel free to change to a name more appropriate,  let's include this unit as an option in the follow up task/issue as well